app/src/main/java/org/oppia/app/player/audio/AudioViewModel.kt

```diff
@@ -96,7 +96,6 @@ class AudioViewModel @Inject constructor(
           languages.first()
         }
         setAudioLanguageCode(languageCode)
-        (fragment as LanguageInterface).languageSelectionClicked()
```

Comment:
Why can this be removed now?

Reply:
languageSelectionClicked() is triggered when the user click(on tvAudioLanguage textview) to change the already selected language. While loading the audio we need not call languageSelectionClicked()  and display the dialog of selecting language in all cases.

Reply:
@nikitamarysolomanpvt Your explanation is correct and therefore we won't need this code here. Thanks.

Reply:
Resolving on behalf of Ben.